lib/packwerk/application_load_paths.rb

```diff
@@ -54,6 +49,19 @@ def relative_path_strings(paths, rails_root: Rails.root)
           .uniq
       end
 
+      private
+
+      sig { params(root: String).void }
+      def assert_application_booted(root)
```

Comment:
The name and implementation of this method seem a bit mismatched now. What do you think of something like this?

```ruby
  def require_application(root)
```

Reply:
Done